specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/networkSecurityPerimeter.json

```diff
@@ -0,0 +1,426 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-01",
+    "title": "StorageManagementClient",
+    "description": "The Azure Storage Management API."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/networkSecurityPerimeterConfigurations": {
+      "get": {
+        "tags": [
+          "StorageAccounts NetworkSecurityPerimeterConfigurations"
+        ],
+        "operationId": "NetworkSecurityPerimeterConfigurations_List",
+        "x-ms-examples": {
+          "NamspaceNetworkSecurityPerimeterConfigurationList": {
+            "$ref": "./examples/NetworkSecurityPerimeterConfigurationList.json"
+          }
+        },
+        "description": "Gets list of effective NetworkSecurityPerimeterConfiguration for storage account",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=69&length=2)

use v3 type please everywhere in this PR

Reply:
corrected.